float64 lx
float64 ly
float64 lz
float64 ox 
float64 oy 
float64 oz

---

float64[] veloutput
